orchagent/neighorch.cpp

```diff
@@ -317,6 +354,14 @@ void NeighOrch::doTask(Consumer &consumer)
             continue;
         }
 
+        if(gPortsOrch->isInbandPort(alias))
```

Comment:
Check for the inband  port type. This is applicable only for the "port" type inband interface